registry/ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "owner": "ed25519_pk1n6xnj04qgs998637uwf2r4tmmp4qlgumjzyw0vz5dxzasskmnsdqpu0l0a",
+  "name": "Zimzalabim",
+  "ticker": "ZIMZA",
+  "homepage": "https://cardanofoundation.org",

```

Comment:
I suggest you to change this.

Reply:
hey mmahut I am unsure what part is the issue in the above code.

Reply:
> I suggest you to change this.

hey mmahut I am unsure what part is the issue in the above code.

Reply:
This looks like your pool would be run by CF.